README.md

```diff
@@ -37,20 +37,21 @@ sudo apt-get install -y llvm clang clang-format libclang-dev
+| Configuration file   | `-c`       | `--config`         | `codegen.json`    | Configuration file to use. Contains all codegen steps to execute, which files to process and with which templates.                                      |
+| Cache file           | `-C`       | `--cache`          | `cache.json`      | Cache file to use. Hash of output and template files will be written to this file to skip future generation if unnecessary.                             |
+| Include directories  | `-I`       | `--includes`       | Empty             | List of include directories to be passed to `libclang`. Used for AST parsing.                                                                           |
+| Macro definitions    | `-D`       | `--definitions`    | Empty             | Macro definitions to be passed to `libclang`. Used for AST preprocessing. Can either be passed as `DEFINITION` or `DEFINITION=VALUE`.                   |
+| Features             | `-f`       | `--features`       | Empty             | Feature flags to be passed to `libclang`. Used for AST preprocessing. **Disabled for now because of some integration issues.**                          |
+| User data            | `-d`       | `--user-data`      | Empty             | Additional user data to be passed to the rendering stage. Can be passed as `key=value` and will be accessible through the `user_data` JSON property.    |
+| C++ standard         | `-s`       | `--cpp-standard`   | `c++14`           | C++ standard to be passed to `libclang`. Can be prefixed with `c++` or not (e.g. `c++17` or `17`).                                                      |
+| Reformat command     | `-r`       | `--reformat`       | `clang-format -i` | Set the command to execute to reformat output files. The command will be supplied a single argument which will be the relative path to the output file. |
+| Force generate       | `-F`       | `--force`          | `false`           | Skip cache and force generate all input files.                                                                                                          |
+| Debug mode           | `-g`       | `--debug`          | `false`           | Enable debug logs.                                                                                                                                      |
+| Sequential mode      | `-S`       | `--sequential`     | `false`           | Enable sequential processing instead of the default parallel processing.                                                                                |
+| Template directories | `-T`       | `--template-paths` | Empty             | List of directories in which to search for templates in case the template is not found in the command's WORKING_DIRECTORY                               |
```

Comment:
I think you're missing a `rebase` on `main`, I don't see your new option `--dump-ast` anymore.

Reply:
ok that was very messy and required a lot of "rebase --continue" and merging but I think it's ok, can you please check?

Reply:
I think you have the same issue in both pull requests. It looks like there was a weird merge. When it's rebased correctly, it only shows the changes you're introducing. Did you rebase from `origin/main` or from the `main` of your fork? I think you need to do something like `git checkout -B main origin/main && git pull` before rebasing unto your branch.